### PR TITLE
feat: Update linkedin help for the new OIDC provider

### PR DESCRIPTION
**Pull request by fragoulis**
Fixes USR-357

### CODE REVIEW

**Comment by fragoulis** [comment]
@alexisintech If you would be kind enough to help me with the 2 new assets I added. The need to have highlights and remove some info. Which tool do you use?

File: docs/authentication/social-connections/linkedin-oidc.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+First, you need to create a new OAuth LinkedIn app.
+
+![Creating an OAuth LinkedIn app](/docs/images/authentication-providers/linkedin/148540a6928a19c42d105d50c91bd4a61c1d3326-1096x978.png)
+
+You need to set a name, associate a LinkedIn page with it, and upload a logo for your new application. Go to the [Social Connections page](https://dashboard.clerk.dev/last-active?path=authentication/social) in your Clerk Dashboard and enable LinkedIn. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Paste the value into the **Redirect URL** of your LinkedIn app, as shown below.
+
+![Obtaining the Application ID and Client secret](/docs/images/authentication-providers/linkedin/linkedin1.png)
+
+The last thing you have to do is to enable the * Sign In with LinkedIn using OpenID Connect* product for your OAuth application.
```
Comment (kostaspt) [inline docs/authentication/social-connections/linkedin-oidc.mdx:35]: This should change to make it bold.

```suggestion
The last thing you have to do is to enable the **Sign In with LinkedIn using OpenID Connect** product for your OAuth application.
```

Comment (kostaspt) [reply docs/authentication/social-connections/linkedin-oidc.mdx:35]: 🙃 Also, maybe change this to "step"?

```suggestion
The last step you have to do is to enable the * Sign In with LinkedIn using OpenID Connect* product for your OAuth application.
```

Comment (fragoulis) [reply docs/authentication/social-connections/linkedin-oidc.mdx:35]: Both of these are actually copy pasted from the deprecated page. I will let someone else say whether I should change these or not.

File: docs/authentication/social-connections/linkedin.mdx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: LinkedIn
+title: LinkedIn (Deprecated)
 description: Learn how to set up social connection with LinkedIn.
 ---
 
 # LinkedIn
```
Comment (alexisintech) [inline docs/authentication/social-connections/linkedin.mdx:6]:
```suggestion
# LinkedIn (deprecated)

```

File: docs/authentication/social-connections/overview.mdx [hunk 3]
```diff
@@ -67,10 +67,18 @@ Clerk provides a wide range of application providers to ease your user's sign-up
       hideArrow={true}
       icon="/docs/images/logos/auth_providers/gitlab.svg"
     />
+    <AuthProvidersCards
+      title="LinkedIn (Deprecated)"
+      description="Add LinkedIn (Deprecated) as an authentication provider for your Clerk app."
+      link="/docs/authentication/social-connections/linkedin"
+      cta="Learn more"
+      hideArrow={true}
+      icon="/docs/images/logos/auth_providers/linkedin.svg"
```
Comment (alexisintech) [inline docs/authentication/social-connections/overview.mdx:76]: if it's deprecated, I don't think we should have a card for it.

```suggestion
```

File: docs/references/javascript/types/oauth-strategy.mdx [hunk 4]
```diff
@@ -65,6 +66,10 @@ type OAuthStrategy = |
     ]}, {cells: [
         <code>oauth_linkedin</code>,
         <code>string</code>,
+        <>Specify LinkedIn (Deprecated) as the verification OAuth provider.</>
```
Comment (alexisintech) [inline docs/references/javascript/types/oauth-strategy.mdx:69]:
```suggestion
        <>This strategy is now deprecated in favor of `oauth_linkedin_oidc`.</>
```

File: public/images/authentication-providers/linkedin/linkedin2.png
Comment (alexisintech) [inline public/images/authentication-providers/linkedin/linkedin2.png:1]: Can we add a red box around the option we are focusing on? Maybe a red box around the correct tab too (the 'Products' tab).

File: public/images/authentication-providers/linkedin/linkedin1.png
Comment (alexisintech) [inline public/images/authentication-providers/linkedin/linkedin1.png:1]: Maybe we can add a red box around the 'Auth' tab to emphasize which tab the user should navigate to.

File: docs/authentication/social-connections/linkedin-oidc.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+How to set up social connection with LinkedIn OpenID Connect
+
+## Overview
+
+Adding social connection with LinkedIn to your app with Clerk is done in a few steps - you only need to set the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+
+To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
+
+For production instances, you will need to generate your own Client ID and Client secret using your LinkedIn account.
```
Comment (alexisintech) [inline docs/authentication/social-connections/linkedin-oidc.mdx:16]:
```suggestion
Learn how to set up a social connection with LinkedIn in your Clerk application.

To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed. Simply navigate to the [Social Connections page](https://dashboard.clerk.dev/last-active?path=user-authentication/social-connections) in your Clerk Dashboard, toggle on the LinkedIn option, and click **Save**.

In production instances, you must provide custom credentials, which includes generating your own Client ID and Client Secret using your LinkedIn Developer account. Don't worry, this guide will walk you through that process in just a few simple steps.
```

File: docs/authentication/social-connections/linkedin-oidc.mdx [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+title: LinkedIn OpenID Connect
+description: Learn how to set up social connection with LinkedIn.
```
Comment (alexisintech) [inline docs/authentication/social-connections/linkedin-oidc.mdx:3]:
```suggestion
description: Learn how to set up a social connection with LinkedIn in your Clerk application.
```

**Review by alexisintech — CHANGES_REQUESTED** [review]

**Review by alexisintech — APPROVED** [review]
😸💖

**Pull request merged**